src/features/game/lib/mapMovement.ts

```diff
@@ -0,0 +1,71 @@
+type MovementIntervals = {
+  moveUp?: any,
+  moveDown?: any,
+  moveLeft?: any,
+  moveRight?: any
+}
+
+
+const sensitivity = 3
+const movementIntervals: MovementIntervals = {}
+const initialCoordinates = [1024,1214]
+// TODO dynamic initial coordinates based on tile dimensions
+
+const setup = (container: any) => {
```

Comment:
Avoid using `any` where possible. Using types reduces bugs and will ensure if someone comes in to change the code there is less of a chance they will break something.

In this case you can type it as `container: RefObject<HTMLDivElement>`

Some other nullable warnings may pop up - this is a good thing, we will need to check if the container is null (could happen if the component unmounts)